Packs/Logzio/Integrations/Logzio/Logzio.py

```diff
@@ -247,6 +262,9 @@ def main():
                              ' Logz.io Security API token, or both.')
         region = demisto.params().get('region')
         first_fetch_time = demisto.params().get('fetch_time', '1 hours')
+        filter_tags = demisto.params().get('tags', None)
+        if filter_tags:
+            filter_tags = filter_tags.split(",")
```

Comment:
should be 
`filter_tags = [tag.strip() for tag in filter_tags.split(",")]`
According to the new documentation :
In the instruction you say "Filter by tags, csv format, for example: tag1, tag2"
Which mean that if customer put more than one tag it will be
"tag1"
" tag2"
And then the filter on tag2 wont work

Reply:
I updated docs and the behavior to be more robust and strip white space.